news/announcing-the-safety-critical-rust-consortium.md

```diff
@@ -0,0 +1,84 @@
+
+The Safety-Critical Rust Consortium intends to keep all stakeholders updated on its work, suggesting modification or additions to how the Rust language and libraries are created and addressing how different safety levels adopt the language and libraries. Doing so will create room for a variety of tooling to support safety activities, verification and validation, and quality assurance within Rust. The work would, in turn, create best practices, which can then be discussed with safety assessors and governments to more officially establish the best practices as recommendations.
+
+## What is the expected impact outside of the Rust Foundation and Project?
+
+Work such as SAE’s would not become obsolete through a Safety-Critical Rust Consortium, it would however become narrower in tuning the wider work to their specific audience, after informing the safety-critical Rust group about their needs. The safety-critical Rust Consortium, and the wider Rust Foundation, would then pursue an industry adoption and advocacy role, with users of Rust as well as with external entities such as governments and international standards groups. The Rust Foundation is committed to ensuring that funding and support are available where needed, including both community volunteers and funded initiatives, such as for this safety initiative.
+
+## Why participate?
+
+In short, your help is needed!
+
+We want the Safety-Critical Rust Consortium to reflect a wide variety of perspectives, including various safety industries, representatives from standards groups guiding these industries, tooling providers who cater to such industries, and Rust community members from the language and library spaces which are interested in collaboration.
+
+### How can I ask another question and/or get involved?
+
+Interested organizations, individuals, and Rust Foundation Members are invited to express interest in joining or submit further questions via email at [contact@rustfoundation.org](contact@rustfoundation.org).
```

Comment:
change to "are invited to get in touch with us via email at contact@rustfoundation.org"